app/Http/Controllers/Api/AirlineController.php

```diff
@@ -3,37 +3,47 @@
 namespace App\Http\Controllers\Api;
 
 use App\Http\Controllers\Controller;
+use App\Http\Requests\GetAirlinesRequest;
 use App\Http\Requests\StoreAirlineRequest;
 use App\Http\Requests\UpdateAirlineRequest;
 use App\Models\Airline;
 use Illuminate\Http\Request;
 
 class AirlineController extends Controller
 {
-    public function index()
+    public function index(GetAirlinesRequest $request)
     {
+        $sortBy = $request->input('sort_by', 'id');
+        $ascending = $request->boolean('asc', true);
+
+        $cityId = $request->input('city');
+        $airlineId = $request->input('flights');
```

Comment:
is this correct? the input is flights but the variable is $airlineId? I believe we can adjust the naming here